server/config/config.go

```diff
@@ -263,6 +277,7 @@ type FleetConfig struct {
 	Upgrades         UpgradesConfig
 	Sentry           SentryConfig
 	GeoIP            GeoIPConfig
+	Prometheus       PrometheusConfig
```

Comment:
Have you checked that yaml properly decoded into the struct? Looking at the structure of the struct, it looks like it would expect a `prometheus.auth.username` and `prometheus.auth.password` configs, but instead it is documented as `prometheus.username` and `prometheus.password`?

Reply:
Good catch, will check this.